hw/top_earlgrey/ip/ast/ast.core

```diff
@@ -9,7 +9,10 @@ filesets:
     depend:
       - lowrisc:ip:tlul
```

Comment:
you can add `- lowrisc:ip:lc_ctrl_pkg` here in order to get the definitions for the life-cycle-related types.

Reply:
There's no lc_ctrl_pkg in v1-bronze. This is abypass I agreed on with @tjaychen. WIll be fixed in SLVR.

Reply:
I agree for `v1-bronze`, but this PR is merged into `master`, right? the package is available on master...

Reply:
@tjaychen just to clarify: should this PR go into `master` or `v1-bronze`?

Reply:
This should go into master.
Once approved I will do another PR for bronze.
I think it is best to have the same ast configuration in both master and bronze-v1. So for now I'd like to keep this file in ast even though it appears elsewhere.
Once lc is added to bronze-v1 I'll remove it.